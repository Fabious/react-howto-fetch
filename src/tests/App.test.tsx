import App from '../App';
import renderer from 'react-test-renderer';

it('smoke', () => {
  const component = renderer.create(<App />);
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
