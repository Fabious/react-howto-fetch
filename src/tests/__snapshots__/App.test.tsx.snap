// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`smoke 1`] = `
<button
  onClick={[Function]}
  type="button"
>
  count is: 
  0
</button>
`;
